packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -711,28 +661,25 @@ export default {
                         timeout: this.checkoutTimeout
                     });
                 }
-            } catch (e) {
-                this.logInvoker('Geo Location Lookup Failed', this.eventData, this.$logger.logWarn);
+            } catch (error) {
+                this.logInvoker('Geo Location Lookup Failed', this.eventData, this.$logger.logWarn, error);
             }
         },
 
         /**
-         * Emit, log and track the error based on the parameters received.
+         * Emit, log and track the error based on the parameters
+         * encapsulated within the 'error' class.
          * Set the `genericErrorMessage` for the user to see.
          */
-        handleErrorState ({
-            error, messageToDisplay, eventToEmit, logMessage
-        }) {
-            const eventData = {
-                ...this.eventData,
-                error
-            };
+        handleErrorState (error) {
+            const message = this.$t(error.messageKey || 'errorMessages.genericServerError');
```

Comment:
If the `messageKey` has a value but there is no translation for it, this line will return an empty string, I believe.

Reply:
I presume that is an issue on all translations, if we have forgotten to add a translation, e.g. if we have forgotten the 'errorMessages.genericServerError' translation then the same thing. Do you want me to protect this line from that scenario and maybe use a hardcoded string if both translations fail to return a string?

Reply:
What I had in mind was doing the following instead:
`const message = this.$t(error.messageKey) ||  this.$t('errorMessages.genericServerError');`